hack/make/.build-rpm/docker-engine.spec

```diff
@@ -144,6 +144,11 @@ install -p -m 644 contrib/udev/80-docker.rules $RPM_BUILD_ROOT/%{_sysconfdir}/ud
 install -d $RPM_BUILD_ROOT/etc/sysconfig
 install -d $RPM_BUILD_ROOT/%{_initddir}
 
+# TODO(runcom): setup this for deb also? right?
```

Comment:
@vdemeester @justincormack no deb expert here - can you point me to the relevant deb lines I need to take care of?